README.md

```diff
@@ -817,10 +817,13 @@ You may also change the marker for `\PhpSchool\CliMenu\MenuItem\CheckboxItem`:
 <?php
 
 use PhpSchool\CliMenu\Builder\CliMenuBuilder;
+use PhpSchool\CliMenu\Style\CheckboxStyle;
 
 $menu = (new CliMenuBuilder)
-    ->setUncheckedMarker('[○] ')
-    ->setCheckedMarker('[●] ')
+    ->modifyCheckboxStyle(function (CheckboxStyle $style) {
+        $style->setMarkerOff('[○] ')
```

Comment:
I think I prefer `setCheckedMarker` and `setUncheckedMarker. We can address that later.

Reply:
I was hoping to keep the naming scheme identical across Selectable, Checkbox and Radio items.

Reply:
However, since removed all the interfaces it doesn't matter much anymore.

Reply:
I think I'd prefer to keep it more semantic. And I think in the end, we might push this rendering back in to the item.

Reply:
I've got PR for SelectableItem styling ready. Should I hold off on it, or should I submit? The code will follow this PR.